.github/workflows/publish.yml

```diff
@@ -105,3 +105,11 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH }}
         run: npm publish --access public
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
+        if: github.event_name == 'workflow_dispatch'
+        with:
+          files: |
```

Comment:
Are more files needed here?

Reply:
Sorry just noticed I left this from C/P from docs.
Here we must list assets we want to upload as part of the release. I think we could include build artifacts perhaps?

Reply:
Yeah. That's what I was thinking. The stuff in `/dist` I think.